oxide.go

```diff
@@ -473,38 +464,39 @@ func (d *Driver) SetConfigFromFlags(opts drivers.DriverOptions) error {
 	d.AdditionalSSHPublicKeyIDs = opts.StringSlice(flagAdditionalSSHPublicKeyIDs)
 	d.SSHPort = defaultSSHPort
 
-	var err error
-
 	if d.Host == "" {
-		err = errors.Join(err, fmt.Errorf("required option not set: %s", flagHost))
+		return errors.New("required option not set: " + flagHost)
 	}
 
 	if d.Token == "" {
-		err = errors.Join(err, fmt.Errorf("required option not set: %s", flagToken))
+		return errors.New("required option not set: " + flagToken)
 	}
 
 	if d.Project == "" {
-		err = errors.Join(err, fmt.Errorf("required option not set: %s", flagProject))
+		return errors.New("required option not set: " + flagProject)
 	}
 
 	if d.BootDiskImageID == "" {
-		err = errors.Join(err, fmt.Errorf("required option not set: %s", flagBootDiskImageID))
+		return errors.New("required option not set: " + flagBootDiskImageID)
 	}
 
-	return errors.Unwrap(err)
+	return nil
```

Comment:
The usage of `errors.Join` and `errors.Unwrap` here was to give the operator all errors upfront rather than making them trial and error their way through required parameters. For example, not setting the host and token but only getting an error back for not setting the host.